pyaedt/edb_core/edb_data/simulation_setup.py

```diff
@@ -0,0 +1,701 @@
+            for k in keys:
+                value = _parse_value(values[keys.index(k)])
+                setter = None
+                if k in dir(edb_sim_setup_info.SimulationSettings):
+                    setter = edb_sim_setup_info.SimulationSettings
+                elif k in dir(edb_sim_setup_info.SimulationSettings.AdvancedSettings):
+                    setter = edb_sim_setup_info.SimulationSettings.AdvancedSettings
+
+                elif k in dir(edb_sim_setup_info.SimulationSettings.DCAdvancedSettings):
+                    setter = edb_sim_setup_info.SimulationSettings.DCAdvancedSettings
+                elif "DCIRSettings" in dir(edb_sim_setup_info.SimulationSettings) and k in dir(
+                    edb_sim_setup_info.SimulationSettings.DCIRSettings
+                ):
+                    setter = edb_sim_setup_info.SimulationSettings.DCIRSettings
+                elif k in dir(edb_sim_setup_info.SimulationSettings.DCSettings):
+                    setter = edb_sim_setup_info.SimulationSettings.DCSettings
+                elif k in dir(edb_sim_setup_info.SimulationSettings.AdvancedSettings):
+                    setter = edb_sim_setup_info.SimulationSettings.AdvancedSettings
+                if setter:
+                    try:
+                        setter.__setattr__(k, value)
+                    except TypeError:
+                        try:
+                            setter.__setattr__(k, str(value))
+                        except:
+                            pass
+            return edb_sim_setup_info
+        elif self._setup_type == "kHFSS":
+            return edb_setup.GetSimSetupInfo()
+
+    @property
+    def enabled(self):
+        """Whether the setup is enabled."""
+        return self._edb_object.SimulationSettings.Enabled
+
+    @enabled.setter
+    def enabled(self, value):
+        self._edb_object.SimulationSettings.Enabled = value
+        self._update_setup()
+
+    @property
+    def name(self):
+        """Name of the Setup."""
+        return self._edb_object.Name
+
+    @name.setter
+    def name(self, value):
+        self._pedb.layout.cell.DeleteSimulationSetup(self.name)
+        self._edb_object.Name = value
+        self._name = value
+        self._update_setup()
+
+    @property
+    def position(self):
+        """Position in the setup list."""
+        return self._edb_object.Position
+
+    @position.setter
+    def position(self, value):
+        self._edb_object.Position = value
+
+    @property
+    def setup_type(self):
+        """Type of the setup."""
+        return self._edb_object.SimSetupType.ToString()
+
+    @property
+    def frequency_sweeps(self):
+        """list of frequency sweep."""
+        temp = {}
+        for i in list(self._edb_object.SweepDataList):
+            temp[i.Name] = EdbFrequencySweep(self, None, i.Name, i)
+        return temp
+
+    @pyaedt_function_handler
+    def _create(self, name=None):
+        """Create a new setup."""
```

Comment:
```suggestion
        """Create a setup."""
```
"Create" implies "new" setup.